dbms/src/Functions/FunctionsGrouping.h

```diff
@@ -41,7 +41,7 @@ extern const int TOO_MANY_ARGUMENTS_FOR_FUNCTION;
     return (num & (num - 1)) == 0;
 }
 
-using ResultType = UInt8;
+using ResultType = UInt64;
 
 class FunctionGrouping : public IFunctionBase
```

Comment:
In `processGroupingIDs` function, it should receive a vector parameter for grouping ids.

Reply:
not really, as commented above,
grouping(a,b,c,xxxx, n) after tidb's rewriting, it changed as grouping(gid) [meta1, meta2, ... metan], it grouping id column is still exactly one, while we should do multi computation between gid and metaX, then get the final result.